crates/accelerate/src/remove_identity_equiv.rs

```diff
@@ -14,6 +14,7 @@ use num_complex::ComplexFloat;
 use pyo3::prelude::*;
 use rustworkx_core::petgraph::stable_graph::NodeIndex;
 
+use crate::commutation_checker::rotation_trace_and_dim;
```

Comment:
Do you think we should put this somewhere more central?

Reply:
Yeah I was looking for a more central place but didn't find an existing one that suited well.. and I didn't want to create a new one just with this file. Maybe we can just create a `gate_metrics.rs` or something that also includes the `gate_fidelity` function